modules/apps/jax-rs/mdm-admin/src/main/java/org/wso2/carbon/mdm/api/User.java

```diff
@@ -402,9 +402,10 @@ public Response getAllUsers() throws MDMAPIException {
             log.debug("Getting the list of users with all user-related information");
         }
         UserStoreManager userStoreManager = MDMAPIUtils.getUserStoreManager();
-        ArrayList<UserWrapper> userList = new ArrayList<UserWrapper>();
+        ArrayList<UserWrapper> userList;
```

Comment:
Recommended fixes can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/166